applications/GeoMechanicsApplication/custom_strategies/schemes/newmark_quasistatic_U_Pw_scheme.hpp

```diff
@@ -494,6 +494,7 @@ class NewmarkQuasistaticUPwScheme : public Scheme<TSparseSpace,TDenseSpace>
         mBeta = 0.25;
         mGamma = 0.5;
         mTheta = 0.5;
+        mDeltaTime = std::numeric_limits<double>::epsilon();
```

Comment:
If this is to suppress the initialization warning I'd just set it to zero. If not, please ignore my comment.